fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -225,6 +227,8 @@ private String asString(final Resource resource, final Map<String, String> value
                 throw new InvalidResourceIdentifierException("Path contains empty element! " + path);
             }
             return path;
+        } else if (isRootWithoutTrailingSlash(resource)) {
```

Comment:
Oh, I see. Sure, the `else` can be removed. I don't have a particular preference on that style.